### DISCUSSION TITLE
Pydantic Field Description

### DISCUSSION THREAD

**Discussion by olliethomas** [mit-ll-responsible-ai/hydra-zen#719] [Q&A] [answered]
Hi, I'm using hydra-zen (which is great btw) and defining my config classes as Pydantic BaseModels. Is there any way to make it so that the `description` argument of `pydantic.Field` propagates to the help message shown when I call my app from the command line with the `--help` flag?

e.g. 
```python
class Config(BaseModel):
    batch_size: Annotated[
        int, Field(description="The number of samples per model training step")
    ] = 256
```

currently produces the help message
```
settings:
  _target_: __main__.Config
  batch_size: 256
```

Is there any way to make is so that something like 
```
settings:
  _target_: __main__.Config
  batch_size: 256 # The number of samples per model training step
```
is produced?

**Comment by rsokl** [comment, marked as answer]
At this point hydra-zen does not control anything about what gets displayed at the CLI. This is all Hydra. hydra-zen may start to take charge here as Hydra is no longer actively maintained, but I have not been planning on taking on this feature in the near term.
